tlatools/org.lamport.tlatools/src/recomp/RVStrategy.java

```diff
@@ -0,0 +1,51 @@
+package recomp;
+
+import java.util.List;
+
+public class RVStrategy implements Runnable {
```

Comment:
Please add some comments to the class, member vars, methods, etc. that describe the purpose of this class and what's going on